app/src/main/java/fr/neamar/kiss/result/AppResult.java

```diff
@@ -47,6 +52,9 @@ public View display(final Context context, int position, View v) {
         TextView appName = (TextView) v.findViewById(R.id.item_app_name);
         appName.setText(enrichText(appPojo.displayName));
 
+        TextView tagsView = (TextView) v.findViewById(R.id.item_app_tag);
+        tagsView.setText(enrichText(appPojo.displayTags));
```

Comment:
If an application has unreadable obscure (or foreign) name and nondescriptive picture, tags can serve as manually assigned name that show what is this app about.

There may be option to hide tags, but there should be a mode that makes tags ever-visible.